package.json

```diff
@@ -197,6 +197,7 @@
 		"jest": "29.5.0",
 		"jest-jasmine2": "29.5.0",
 		"jest-junit": "13.0.0",
+		"jest-matchmedia-mock": "1.1.0",
```

Comment:
I added this library hoping to emulate event-driven but kept it for simplicity. Happy to remove and restore old mock methods.